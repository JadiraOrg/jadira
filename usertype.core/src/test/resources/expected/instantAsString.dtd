<!ELEMENT dataset (
    INSTANTASSTRING*)>

<!ELEMENT INSTANTASSTRING EMPTY>
<!ATTLIST INSTANTASSTRING
    ID CDATA #REQUIRED
    INSTANT CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

