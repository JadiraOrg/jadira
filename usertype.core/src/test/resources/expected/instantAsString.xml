<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE dataset SYSTEM "instantAsString.dtd">
<dataset>
  <INSTANTASSTRING ID="0" INSTANT="2004-02-25T17:03:45.760Z" NAME="test_0"/>
  <INSTANTASSTRING ID="1" INSTANT="1980-03-11T00:03:45Z" NAME="test_1"/>
</dataset>